railties/lib/rails/generators/app_base.rb

```diff
@@ -378,13 +378,17 @@ def bundle_command(command, env = {})
         _bundle_command = Gem.bin_path("bundler", "bundle")
```

Comment:
Please remove the above `We unset temporary bundler variables to load proper bundler and Gemfile.
` comment.

Reply:
Comment removed.